pages/report.tsx

```diff
@@ -75,15 +77,44 @@ function useInitialFormValues(): UnsubmittedFormResponse {
     email,
     should_subscribe: false,
     confirmed_get_app_updates: false,
+    image_uuid: formUuid,
   };
 }
 
 export default function Report() {
   const [currentStep, setCurrentStep] = useState(FormStep.SightingDetails);
   const { session } = useSessionUser();
-  const defaultFormFormValues = useInitialFormValues();
+  const [formUUID] = useState(uuidv4());
+  const defaultFormFormValues = useInitialFormValues(formUUID);
   const submitForm = useSubmitSharkSightingForm();
 
+  const splitFileReturnExtension = (fileName: string) => {
+    return fileName.split(".").pop();
+  };
+
+  const onFileChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const fileList = e.target.files;
+    if (!fileList) return;
+    for (let i = 0; i < fileList.length; i++) {
+      // 5242880 = 5mb
+      if (!fileList[i] || fileList[i].size > 5242880) return;
```

Comment:
Should render this as an error to the user so they don't get confused (toast would good)!

Is this sufficient also to remove the file from the uploaded files list in the `<input>`?